src/bokeh_fastapi/handler.py

```diff
@@ -361,3 +361,6 @@ async def send_message(self, message: Message) -> int:
         except WebSocketDisconnect as e:
             self.on_close(e.code, e.reason)
         return sent
+
+    def ping(self, data: bytes | str = b"") -> None:
+        raise NotImplementedError
```

Comment:
This is needed by the keep-alive schedule. @philippjfr since you are way more familiar with WS and the corresponding protocol, could you implement this?

Reply:
Will do.

Reply:
Turns out this is handled by `uvicorn` itself.